.pre-commit-config.yaml

```diff
@@ -7,37 +7,15 @@ repos:
     -   id: end-of-file-fixer
     -   id: trailing-whitespace
     -   id: mixed-line-ending
--   repo: https://github.com/pycqa/flake8
-    rev: 6.0.0
-    hooks:
-    -   id: flake8
 -   repo: https://github.com/pycqa/doc8
     rev: v1.1.1
     hooks:
     -   id: doc8
--   repo: https://github.com/asottile/pyupgrade
-    rev: v3.4.0
-    hooks:
-    -   id: pyupgrade
-        args: [--py38-plus]
 -   repo: https://github.com/adamchainz/django-upgrade
     rev: 1.13.0
     hooks:
     -   id: django-upgrade
         args: [--target-version, "3.2"]
--   repo: https://github.com/pycqa/isort
-    rev: 5.12.0
-    hooks:
-    -   id: isort
--   repo: https://github.com/pre-commit/pygrep-hooks
-    rev: v1.10.0
-    hooks:
-    -   id: python-check-blanket-noqa
-    -   id: python-check-mock-methods
-    -   id: python-no-eval
-    -   id: python-no-log-warn
-    -   id: rst-backticks
-    -   id: rst-directive-colons
```

Comment:
ruff doesn't handle these two hooks. Is there a good reason for removing them?

Reply:
My bad, I had the impression that ruff covered all of them. Fixed.